cg/constants/encryption.py

```diff
@@ -56,3 +56,9 @@ class GPGParameters(ListEnum):
     OUTPUT_PARAMETER: list = [
         "-o",
     ]
+
+
+class EncryptionDirsAndFiles(StrEnum):
+    """Encryption related dirs and files"""
+
+    ENCRYPT_DIR: str = "/home/hiseq.clinical/ENCRYPT/"
```

Comment:
this shouldnt be hardcoded in cg, but come from config file in servers or envvars

Reply:
Created issue with this comment: https://github.com/Clinical-Genomics/cg/issues/1515